artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQServerLogger.java

```diff
@@ -719,7 +719,7 @@ void slowConsumerDetected(String sessionID,
    @Message(id = 222094, value = "Bridge unable to send message {0}, will try again once bridge reconnects", format = Message.Format.MESSAGE_FORMAT)
    void bridgeUnableToSendMessage(@Cause Exception e, MessageReference ref);
 
-   @LogMessage(level = Logger.Level.WARN)
+   @LogMessage(level = Logger.Level.INFO)
```

Comment:
What is the difference? just the [WARN] on the logs?

Why is that an issue?


Shouldn't the proper fix be.. if it's shutting down.. log.debug, if !... log.warn?


or this is about client nodes?